client/app/styles/onboarding.styl

```diff
@@ -27,13 +33,24 @@ header
             display: block
 
         svg
-            max-width: 100%
-            height:    (200 / 16)em
+            width:          4.5em
+            height:         4.5em
+            border-radius:  50%
+            padding:        1.5em
+            box-shadow:     0 0 1em 0 #ccc
 
     h1
         margin: 0
 
 
+figure[data-type='preset']
```

Comment:
I don't think a `data-type` attribute is useful, it's too specific IMO. We may try to avoid them, and prefer use and `id` if we really need to target a specific isolated element.